commands/create.go

```diff
@@ -4,22 +4,39 @@ import (
 	"fmt"
 	"github.com/jfrog/jfrog-client-go/utils/log"
 	"net/http"
+	"time"
 )
 
 type bundleID string
 
 type createSupportBundleHTTPClient interface {
 	GetURL() string
-	CreateSupportBundle(payload string) (int, []byte, error)
+	CreateSupportBundle(payload SupportBundleCreationOptions) (int, []byte, error)
+}
+
+type optionsProvider interface {
+	GetOptions(caseNumber string) (SupportBundleCreationOptions, error)
+}
+
+type defaultOptionsProvider struct {
+	getDate func() time.Time
+}
+
+func (p *defaultOptionsProvider) GetOptions(caseNumber string) (SupportBundleCreationOptions, error) {
+	return SupportBundleCreationOptions{
+		Name:        fmt.Sprintf("JFrog Support Case number %s", caseNumber),
+		Description: fmt.Sprintf("Generated on %s", p.getDate().Format(time.RFC3339)),
```

Comment:
we should make sure we are using UTC everywhere, and the same format (`time.RFC3339)`